src/components/common/ContactInfo.vue

```diff
@@ -2,17 +2,17 @@
   <ul class="contact-info pl-0">
     <li class="contact-container">
       <v-icon class="contact-icon" size="18px">mdi-phone</v-icon>
-      <span class="pr-2">Canada and U.S. Toll Free:</span>
+      <span class="px-2">Canada and U.S. Toll Free:</span>
```

Comment:
A couple of small changes requested by Tracey: more whitespace before the text here, and some font tweaks below.

Reply:
(I tested in detail components and a couple of dialogs and it looks good.)

Reply:
In a detail component:

![image](https://user-images.githubusercontent.com/10002889/140543700-f2fde8e9-f660-42a5-b2c8-a860760a252d.png)

Reply:
In a dialog:

![image](https://user-images.githubusercontent.com/10002889/140543891-77f65e69-a75c-4ea4-9f94-c41c9a689a51.png)

Reply:
In a dialog:

![dialog with 15px body text](https://user-images.githubusercontent.com/10002889/140564261-80817501-7bfa-4113-9421-6fca34a9d39a.png)

^^^ I updated this dialog with the 15px body text (instead 14.4px as before). This was OKed by Tracey.